src/templates/kit/screens/DotVotingScreen.js

```diff
@@ -0,0 +1,295 @@
+        justify-content: center;
+      `}
+    >
+      <Header
+        title="Configure template"
+        subtitle={
+          <span
+            css={`
+              display: flex;
+              align-items: center;
+              justify-content: center;
+            `}
+          >
+            Choose your
+            <span
+              css={`
+                display: flex;
+                margin: 0 ${1.5 * GU}px;
+              `}
+            >
+              <KnownAppBadge
+                appName="dot-voting.aragonpm.eth"
+                label={appLabel}
+              />
+            </span>
+            settings below.
+          </span>
+        }
+      />
+      <PercentageField
+        ref={handleSupportRef}
+        label={
+          <React.Fragment>
+            Support %
+            <Help hint="What’s the support?">
+              <strong>Support</strong> is the minimum percentage of votes
+              required on a dot voting option for it to be considered valid. For
+              example, if "Support %" is set to 5%, then an option needs at
+              least 5% of the total dot votes to be considered valid.
+            </Help>
+          </React.Fragment>
+        }
+        value={support}
+        onChange={handleSupportChange}
+      />
+      <PercentageField
+        ref={quorumRef}
+        label={
+          <React.Fragment>
+            Minimum Participation %
+            <Help hint="What’s the minimum participation?">
+              <strong>Minimum Participation</strong> is the minimum percentage
+              of the total token supply that is required to participate in a dot
+              vote for the proposal to be considered valid. For example, if
+              "Minimum Participation %" is set to 51%, then at least 51% of the
+              outstanding token supply must have participated in the vote for
+              the vote to be considered valid.
+            </Help>
+          </React.Fragment>
+        }
+        value={quorum}
+        onChange={handleQuorumChange}
+      />
+      <Duration
+        duration={duration}
+        onUpdate={handleDurationChange}
+        label={
+          <React.Fragment>
+            Vote duration
+            <Help hint="What’s the vote duration?">
+              <strong>Vote Duration</strong> is the length of time that the vote
+              will be open for participation. For example, if the Vote Duration
+              is set to 24 hours, then token-holders have 24 hours to
+              participate in the vote.
+            </Help>
+          </React.Fragment>
+        }
+      />
+      {formError && (
+        <Info
+          mode="error"
+          css={`
+            margin-bottom: ${3 * GU}px;
+          `}
+        >
+          {formError}
+        </Info>
+      )}
+      <Info
+        css={`
+          margin-bottom: ${3 * GU}px;
+        `}
+      >
+        The support and minimum participation thresholds are strict
```

Comment:
Just wanted to double check this was true, since it was noted above that quorum was not allowed to be 0 (which suggests the calculation may be greater than or equal to, rather than just strictly greater than).

Reply:
In the contract the calculation is just strictly greater than, but after trying that in the UI it did look ugly to allow values like `1e-18` or even `0.1` because the input component has not enough width.

I commented that with @quazia and we decided a minimum 1% was enough, any alternative suggestion @sohkai ?